fedora/README.md

```diff
@@ -26,7 +26,7 @@ The metalink `http://mirrors.fedoraproject.org` is used to automatically select
 
 # Supported Docker versions
 
-This image is officially supported on Docker version 1.6.2.
+This image is officially supported on Docker version 1.6.2 and later.
```

Comment:
There are newer Docker versions than 1.6.2?  Where? :smile:

(RCs don't really count :+1:, but this part of this file is generated besides :wink:)